runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/client/WindmillStream.java

```diff
@@ -68,21 +70,34 @@ Windmill.KeyedGetDataResponse requestKeyedData(
   /** Interface for streaming CommitWorkRequests to Windmill. */
   @ThreadSafe
   interface CommitWorkStream extends WindmillStream {
+    @NotThreadSafe
+    interface RequestBatcher extends Closeable {
```

Comment:
I think this may need to be AutoCloseable for close() to be automatically called after try with resources block. Or else may need to explicitly call close().

Reply:
Closeable (in io package) extends AutoCloseable https://docs.oracle.com/javase/8/docs/api/java/io/Closeable.html